### PR TITLE
docs: manual pages for `select()` and `mutate()`

### PR DESCRIPTION
**Pull request by maelle**
#214

### CODE REVIEW

File: R/select.R [hunk 1]
```diff
@@ -16,6 +17,9 @@ select.duckplyr_df <- function(.data, ...) {
   rel_try(list(name = "select", x = .data, args = try_list(dots = enquos(...))),
     # We could count and create a zero-col data frame, but we can't
     # create a duckplyr frame from it anyway.
+    #' @section Caveats:
```
Comment (maelle) [inline R/select.R:20]: Maybe this section should be called Fallbacks?

Comment (maelle) [reply R/select.R:20]: Oh and we could have an automatic sentence about telemetry in all fallback section later once telemetry is easier to tweak?

File: R/select-rd.R [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#' @title Keep or drop columns using their names and types
+#'
+#' @description  This is a method for the [`dplyr::select()`] generic.
```
Comment (maelle) [inline R/select-rd.R:3]: also can we say "generic" here

File: R/select.R [hunk 3]
```diff
@@ -16,6 +17,9 @@ select.duckplyr_df <- function(.data, ...) {
...
+    #' You cannot use `select.duckplyr_df` with no expression,
```
Comment (krlmlr) [inline R/select.R:21]: or with a selection that returns no columns

**Comment by krlmlr** [comment]
That won't fly, unfortunately. Looks like if we want to reexport, we need to document.

```
  Undocumented code objects:
    ‘mutate’ ‘select’
```

https://github.com/tidyverse/duckplyr/actions/runs/11953558550/job/33321797657?pr=351#step:17:223

@hadley: Neither dbplyr nor dtplyr reexport the dplyr generics. I'm inclined to avoid reexporting here too, but to support the existing

```r
library(duckplyr)
data.frame(a = 1) |> mutate(b = 2)
```

pattern we'd need to add `Depends: dplyr` . What do you think?

**Comment by krlmlr** [comment]
Let's ignore the reexport question for the purpose of this PR, and focus only on providing `?mutate.duckplyr_df` and similar.

**Comment by hadley** [comment]
Why can't you just document the methods?

**Comment by maelle** [comment]
@krlmlr since the main branch was merged into this I don't think I can rebase. Should I start a new branch or do you want to squash?

**Pull request closed without merging**